keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -327,9 +327,15 @@ def _get_initial_alphas(self, timesteps):
         return alphas, alphas_prev
 
     def _get_initial_diffusion_noise(self, batch_size, seed):
-        return tf.random.normal(
-            (batch_size, self.img_height // 8, self.img_width // 8, 4), seed=seed
-        )
+        if seed:
```

Comment:
if seed is not None (it might be 0 which is a valid seed)

Reply:
Agh thank you for catching this -- done